Packs/CrowdStrikeFalconStreamingV2/Integrations/CrowdStrikeFalconStreamingV2/CrowdStrikeFalconStreamingV2.py

```diff
@@ -299,25 +328,45 @@ async def long_running_loop(
         stream (EventStream): CrowdStrike Falcon stream to fetch events from.
         offset (int): Stream offset to start the fetch from.
         event_type (str): Stream event type to fetch.
-        verify_ssl (bool): Whether the request should verify the SSL certificate.
-        proxy (bool): Whether to run the integration using the system proxy.
+        verify_ssl (bool): Whether the request should verify the SSL certificate or not.
+        proxy (bool): Whether to run the integration using the system proxy or not.
+        incident_type (str): Type of incident to create.
+        store_samples (bool): Whether to store sample events in the integration context or not.
 
     Returns:
         None: No data returned.
     """
+    sample_events_to_store = deque(maxlen=20)  # type: ignore[var-annotated]
+    last_sample_events_storage = datetime.utcnow()
     async with init_refresh_token(base_url, client_id, client_secret, verify_ssl, proxy) as refresh_token:
         stream.set_refresh_token(refresh_token)
-        async for event in stream.fetch_event(offset=offset, event_type=event_type):
+        async for event in stream.fetch_event(initial_offset=offset, event_type=event_type):
             event_metadata = event.get('metadata', {})
             event_type = event_metadata.get('eventType', '')
             event_offset = event_metadata.get('offset', '')
             demisto.info(f'Fetching event with offset: {event_offset}')
             incident_name = f'{event_type} - offset {event_offset}'
+            event_dump = json.dumps(event)
             incident = [{
                 'name': incident_name,
-                'details': json.dumps(event)
+                'details': event_dump,
+                'rawJSON': event_dump,
+                'type': incident_type
             }]
             demisto.createIncidents(incident)
+            integration_context_to_set: Dict[str, Union[int, str]] = {'offset': int(event_offset) + 1}
+            if store_samples:
+                try:
+                    sample_events_to_store.append(event)
+                    if last_sample_events_storage + timedelta(minutes=5) <= datetime.utcnow():
```

Comment:
As we have the `store_samples` param let's reduce this to 1 minute so users don't need to wait too long to get first results.
```suggestion
                    if last_sample_events_storage + timedelta(minutes=1) <= datetime.utcnow():
```